app/browser/reducers/passwordManagerReducer.js

```diff
@@ -189,14 +189,17 @@ const migrate = (state) => {
                                                password.get('authTag'),
                                                masterKey,
                                                password.get('iv'))
-      let form = {}
-      form['origin'] = password.get('origin')
-      form['signon_realm'] = password.get('origin') + '/'
-      form['action'] = password.get('action')
-      form['username'] = password.get('username')
-      form['password'] = decrypted
-      autofill.addLogin(form)
+      if (decrypted) {
+        let form = {}
+        form['origin'] = password.get('origin')
+        form['signon_realm'] = password.get('origin') + '/'
+        form['action'] = password.get('action')
+        form['username'] = password.get('username')
+        form['password'] = decrypted
+        autofill.addLogin(form)
+      }
     })
+    state = state.set('legacy_passwords', state.get('passwords'))
```

Comment:
Thanks!
1. This field should be documented in docs/state.md. 
2. should be `legacyPasswords` instead of legacy_passwords
3. i think you can call this once in https://github.com/brave/browser-laptop/pull/8369/commits/015e9b31ae8f86825100782279d73a29f3279c28#diff-6547e1a0bf9650c63b5500045f5ecee5R185 instead of once per password